plugins/dot/src/main/java/org/qbicc/plugin/dot/DotGenerator.java

```diff
@@ -138,6 +138,11 @@ private void process(final MemberElement element, MethodBody methodBody) {
             failedToWrite(ctxt, path, cause);
         } catch (TooBigException e) {
             log.debugf("Element \"%s\" is too big to graph", element);
+            try {
+                Files.delete(path);
```

Comment:
Some operating systems will not let you delete a file while it is open. I would suggest putting this catch block on a try block that encloses this one to avoid that possibility.